bb/ramfs.go

```diff
@@ -88,46 +110,64 @@ func ramfs() {
 		log.Fatalf("%v\n", err)
 	}
 
-	paths[filepath.Join(config.Gopath, "src/github.com/u-root/u-root/bb/bbsh")] = "init"
+	paths[filepath.Join(config.Gopath, "src/github.com/u-root/u-root/bb/bbsh")] = []string{"init", "ubin"}
 
-	// For now, just one added path.
 	if *extraPaths != "" {
-		x := strings.Split(*extraPaths, ":")
-		if len(x) != 2 {
-			log.Fatalf("extraPaths requires two : separated pathnames")
+		extras := strings.Split(*extraPaths, " ")
+		for _, x := range extras {
+			paths["/"] = append(paths["/"], x)
```

Comment:
The : thing is gone, it was not really useful, at least I never found a need for it, and I suspected it would only confuse people. If you'd like it back however I can bring it back, it's quite easy to do.
So the 'extra' arguments now just get put into the paths at ["/"] and processed in that way.